BTrees/tests/testBTreesUnicode.py

```diff
@@ -29,33 +29,31 @@ def setUp(self):
         #setup an OOBTree with some unicode strings
         from BTrees.OOBTree import OOBTree
         from BTrees._compat import _bytes
-        from BTrees._compat import _u
 
-        self.s = _u(b'dreit\xe4gigen', 'latin1')
+        self.s = b'dreit\xe4gigen'.decode('latin1')
```

Comment:
This is equivalent to `u'dreit\xe4gigen'`.

I expect there was an explicit `latin1` because `_u('...\xe4...')` did not work.

Reply:
So it is!